R/print.data.table.R

```diff
@@ -7,6 +7,7 @@ print.data.table = function(x, topn=getOption("datatable.print.topn"),
                col.names=getOption("datatable.print.colnames"),
                print.keys=getOption("datatable.print.keys"),
                trunc.cols=getOption("datatable.print.trunc.cols"),
+               print.indices=getOption("datatable.print.indices"),
```

Comment:
Hmm, this new name is worse IMO, because `print.keys` already handles printing the `Key:` and `Indices:` summaries above the table. WDYT about `show.indices` or even `show.indices.values`/`show.index.values`?

Reply:
I think I prefer `show.indices`, just because I think something like `show.index.values` might be a bit wordy to read if the user sets it as an option, ie `options(datatable.show.index.values=TRUE)`